### DISCUSSION TITLE
Welcome to radiowecker Discussions!

### DISCUSSION THREAD

**Discussion by beabel** [beabel/radiowecker#2] [Ankündigungen / Announcements]
## 👋 Willkommen!
  Wir nutzen Diskussionen als Ort, um mit anderen Mitgliedern unserer Gemeinschaft in Kontakt zu treten. Wir hoffen, dass Sie:
  * Fragen stellen, über die Sie sich wundern.
  * Ideen austauschen.
  * sich mit anderen Community-Mitgliedern austauschen.
  * Andere willkommen heißen und aufgeschlossen sind. Denken Sie daran, dass dies eine Gemeinschaft ist, die wir
  gemeinsam aufbauen 💪.
---------------------------------------------------------
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by DarnokHillstar** [comment]
Die 4.0 Variante ist Klasse! 
Ich hatte an eine ähnliche Idee gedacht mit einem BME280 Sensor mit ESP-NOW. 
So ist es aber auch gut gelöst und passt mir vollkommen.

Danke

Darnok
